rubocop-airbnb/rubocop-airbnb.gemspec

```diff
@@ -25,9 +25,9 @@ Gem::Specification.new do |spec|
     'Gemfile',
   ]
 
-  spec.add_dependency('rubocop', '~> 0.93.1')
+  spec.add_dependency('rubocop', '~> 1.22.0')
```

Comment:
I decided to bump the version to grater than 1.22.0 because if I tried running rubocop with any version between 1.0.0 and 1.21.0 errors would occur, indicating the following message: 

```
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop/rubocop/issues
```